gui/js/lod_body_settings.js

```diff
@@ -11,12 +11,3 @@ $('#set_enable_auto_terrain_lod').change(function () {
     else
         $("#set_terrain_lod").removeClass("unresponsive");
 });
-
-
-function set_map_data_copyright(text) {
```

Comment:
These methods have to be in a `lodBodiesApi`.

Reply:
Done in https://github.com/cosmoscout/csp-lod-bodies/pull/8/commits/a843e3046720f24bc848389587ee4b1dec24eea7